Python/instrumentation.c

```diff
@@ -16,14 +16,14 @@
 
 static PyObject DISABLE =
 {
-    _PyObject_IMMORTAL_REFCNT,
-    &PyBaseObject_Type
+    .ob_refcnt = _PyObject_IMMORTAL_REFCNT,
```

Comment:
Explicit setting of these fields seems a bit odd to me. 
@markshannon should review and merge it since this is recently introduced file.